mne/viz/_3d.py

```diff
@@ -1680,6 +1680,7 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
 
     time_label, times = _handle_time(time_label, time_unit, stc.times)
     # convert control points to locations in colormap
+    user_colormap = colormap  # save the original colormap
```

Comment:
I thought that moving this line down *below* the call to `_limits_to_control_points` would fix the colormap-switching behavior, but it didn't (it did the weird positive-only limits for `mne` colormap` behavior). I can live with the current iteration, then